sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -12,7 +12,7 @@ Use the client library for App Configuration to create and manage application co
 
 ### Prerequisites
 
-- Java Development Kit (JDK) with version 8 or above
+- [Java Development Kit (JDK)][azure_jdk] with version 8 or above
```

Comment:
We should be compatible with any distribution of JDK 8+ and not just `azure_jdk`. Is there a reason for linking to Azure JDK distribution here?

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/10320

First section suggestted.